src/components/templates/FireBarrel/FireBarrel.styled.ts

```diff
@@ -6,6 +6,12 @@ export const Wrapper = styled.div`
   margin: 6rem auto;
   grid-template-columns: repeat(4, 1fr);
   grid-gap: 1rem;
+
+  @media (min-width: 1200px) {
+    width: calc(100% - 360px);
+    margin: 0;
+    padding: 3rem;
+  }
```

Comment:
Given `src/components/organisms/WithNavigationBar/WithNavigationBar.scss` sets `flex` styles, i'm curious why we need to do anything 'special' here? Shouldn't this 'inherit' the width it can use based on that? There's a possibility we might need to use `min-width` and/or `max-width`, to make that work.. but I expect we shouldn't need to be doing anything 'special' to calculate the size here ourselves.

Reply:
fixed by `LARGE_SCREEN_WIDTH `